master/internal/experiment/bulk_action.go

```diff
@@ -60,6 +54,18 @@ func terminalExperimentStates() []string {
 	return states
 }
 
+func nonTerminalStates() []experimentv1.State {
+	states := make([]experimentv1.State, len(experimentv1.State_value)-len(model.TerminalStates))
```

Comment:
two tips.
1, there really isn't enough benefit for doing this when it makes the code more difficult to read, imo, unless it is called _all the time_
2, you can do `make([]experimentv1.State, 0, x)` to make a slice with capacity = x but length 0 so you get all the perf benefits of what you are trying to do (preallocating the correct size backing array) without needing to use an `idx` (it is 0 length, so you can just append)

Reply:
switched back to append